migrations/20230704145116-create_users_table.sql

```diff
@@ -0,0 +1,10 @@
+
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS users (
+    user_id SERIAL PRIMARY KEY,
+    github_username TEXT,
+    github_ID INTEGER,
+    UNIQUE (user_id)
+);
+-- +migrate Down
```

Comment:
Let us add the migrate down (rollback) section to reverse these changes, for good practice.